master/release-release-v1alpha1.json

```diff
@@ -2,17 +2,17 @@
   "description": "Release is a Kubernetes resource (CR) representing a Giant Swarm workload cluster release.",
   "properties": {
     "apiVersion": {
-      "description": "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
+      "description": "APIVersion defines the versioned schema of this representation of an object.\nServers should convert recognized schemas to the latest internal value, and\nmay reject unrecognized values.\nMore info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources",
       "type": "string"
     },
     "kind": {
-      "description": "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
+      "description": "Kind is a string value representing the REST resource this object represents.\nServers may infer this from the endpoint the client submits requests to.\nCannot be updated.\nIn CamelCase.\nMore info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds",
       "type": "string"
     },
     "metadata": {
       "properties": {
         "name": {
-          "pattern": "^v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(-[\\.0-9a-zA-Z]*)?$",
+          "pattern": "^(((aws|azure|vsphere|cloud-director|eks)-)|v)(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(-[\\.0-9a-zA-Z]*)?$",
```

Comment:
What is this exactly? I'm surprised not to see `capa` here.

Reply:
Oh, ignore me, `capa` is only used in the directory structure, not the release.yaml name 🤦  Not at all confusing.

Reply:
Oh actually, looks like the `v` is meant to be optional though? e.g. https://github.com/giantswarm/releases/blob/3bc4ab8a804fffff626d121d17bae62dee5ee23e/capa/v29.1.0/release.yaml#L4

Reply:
Yeah, I think my last comment still stands though right?

Reply:
if I read that pattern right, it's either the `provider` prefix or the `v`. So it's not optional, but it's either one or the other.

Reply:
Oh gotcha! Regex fun! :p